.env

```
@@ -116,3 +117,12 @@ FILESYSTEM_USER_FILES_ADAPTER_S3_BUCKET=test
 FILESYSTEM_USER_FILES_ADAPTER_S3_PREFIX=
 
 RECREATE_QA_USERS=false
+
+###> rostering file-upload ###
+FILESYSTEM_ROSTERING_FILES_ADAPTER_S3_BUCKET=test
+FILESYSTEM_ROSTERING_FILES_ADAPTER_S3_PREFIX=
+S3_STORAGE_PENDING_FOLDER_NAME=pending
+S3_STORAGE_FAILED_FOLDER_NAME=failed
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The S3_STORAGE_FAILED_FOLDER_NAME key should go before the S3_STORAGE_PENDING_FOLDER_NAME key

(UnorderedKey)